packages/theme/css/characteristics/container.css

```diff
@@ -11,8 +11,8 @@
   --salt-container-secondary-borderColor: var(--salt-palette-neutral-secondary-border);
   --salt-container-secondary-borderColor-disabled: var(--salt-palette-neutral-secondary-border-disabled);
 
-  --salt-container-tertiary-background: var(--salt-palette-neutral-tertiary-background);
```

Comment:
curious why these palette ones were removed if we still have the container tokens of transparent value. did we change all transparent tokens to 'transparent' hardcoded? the original reason we didnt do that is because when the theme is customised it may not necessarily be transparent in a new theme

Reply:
afaik, only `palette-neutral` and `palette-navigate` has the tokens with `transparent` value deprecated. `palette-interact` still has several tokens referencing `transparent` as values. Let me check with Paul re this

Reply:
Update Nov 13: Paul has confirmed that container shouldn't have a tertiary variant any more. Will deprecate the container tertiary tokens too